docs/src/main/asciidoc/amqp-guide.adoc

```diff
@@ -0,0 +1,312 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+
+include::./attributes.adoc[]
+= {project-name} - Using AMQP with Reactive Messaging
```

Comment:
You need to have the title as the very first element (just below the comment with no additional new line), otherwise it now gets ignored by the website.

Obviously, you need to replace the variable with `Quarkus`.

Reply:
wow.. this is weird, ok. Doing it.

Reply:
Yeah, probably due to an update of a component but I didn't have the time to dig more. Respecting this rule fixes the issue.

Reply:
Done anyway.